server/lobbyconnection.py

```diff
@@ -976,7 +980,11 @@ async def command_game_join(self, message):
 
         if not game or game.state is not GameState.LOBBY:
             self._logger.debug("Game not in lobby state: %s state %s", game, game.state)
-            await self.send({
+            await self.send_game_join_failed_response({
```

Comment:
We should probably add an integration test for this case, I'm actually a little surprised there isn't one yet. We should also update any integration tests that are currently checking for the notice messages to check for the `game_join_failed` messages as well.